ui/components/component-library/text-field-base/text-field-base.stories.js

```diff
@@ -144,7 +151,13 @@ export default {
   },
 };
 
-const Template = (args) => <TextFieldBase {...args} />;
+const Template = (args) => {
+  const [{ value }, updateArgs] = useArgs();
+  const handleOnChange = (e) => {
+    updateArgs({ value: e.target.value });
+  };
+  return <TextFieldBase {...args} value={value} onChange={handleOnChange} />;
+};
```

Comment:
This allows for the value prop to work in the component and as a control


https://user-images.githubusercontent.com/8112138/203656571-bfb14167-8877-471d-96aa-e8517b783be9.mov